specification/network/resource-manager/Microsoft.Network/2017-10-01/expressRouteCircuit.json

```diff
@@ -1162,7 +1162,9 @@
         },
         "peerASN": {
           "type": "integer",
-          "format": "int32",
+          "format": "int64",
```

Comment:
If these numbers are four bytes why the change to int64?  I addition this will introduce a breaking change in SDKs.